<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorInterface</class>
 <widget class="QWidget" name="CalculatorInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorInterface</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>256</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>256</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_plus">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>256</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>256</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>285</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>285</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_minus">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>285</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>285</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_0">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>343</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_razy">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_AC">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>256</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Clear">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>227</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_rowna">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_dziel">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>343</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_dwa">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dwojkowy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_osiem">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ã³semkowy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_szesnascie">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>szesnastkowy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_prawo">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>343</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_lewo">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>343</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_dziesiec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dziesietny</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dziesietny">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Mplus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>M+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Mminus">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>M-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Mprzywroc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>MR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Mclear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>MC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_A">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_B">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_C">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_D">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_E">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_F">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
